tests/test_refcoco.py

```diff
@@ -0,0 +1,74 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import json
+import os
+from unittest import TestCase
+
+
+from datasets import Dataset as HFDataset
+from datasets import DatasetDict
+from mmengine.config import Config, ConfigDict
+from PIL import Image
+from torch.utils.data import Dataset
+import copy
+from xtuner.dataset import RefCOCOTrainDataset, InvRefCOCOTrainDataset
+from xtuner.dataset.refcoco import Blip2ImageTrainProcessor
+from xtuner.dataset.map_fns import refcoco_map_fn
+from xtuner.registry import BUILDER
+import torch
+
+
+import logging
+
+PATH = "xtuner/configs/llava/llava_vicuna_7b_v15_clip_vit_large_p14_e1_gpu8_pretrain.py"
+
+
+class TestRef(TestCase):
+
+    def test_ref(self):
+        config_path = PATH
+        dataset_config = Config.fromfile(config_path)['llava_dataset']
+
+        refcoco_dataset_config = copy.copy(dataset_config)
+        refcoco_dataset_config['type'] = RefCOCOTrainDataset
+        refcoco_dataset_config['data_path'] = 'data/refcoco/refcoco_annotations'
+        refcoco_dataset_config['image_folder'] = 'data/refcoco/train2014'
```

Comment:
```suggestion
        refcoco_dataset_config['image_folder'] = 'data/coco_images'
```

Is this modification correct?